server/src/test/proto/spine/test/event/commands.proto

```diff
@@ -24,21 +24,55 @@ package spine.test.event.command;
 import "spine/options.proto";
 
 option (type_url_prefix) = "type.spine.io";
-option java_package="io.spine.test.event.command";
+option java_package = "io.spine.test.event.command";
 option java_outer_classname = "EventCommandsProto";
 option java_multiple_files = true;
 
+import "spine/core/user_id.proto";
+import "spine/net/email_address.proto";
 import "spine/test/event/project.proto";
+import "spine/test/event/team.proto";
 
 message CreateProject {
     ProjectId project_id = 1;
+
+    TeamId team_id = 2;
 }
 
 message AddTask {
     ProjectId project_id = 1;
+
     Task task = 2;
 }
 
+message AddTasks {
+    ProjectId project_id = 1;
+
+    repeated Task task = 2;
+}
+
 message StartProject {
```

Comment:
Please think of a prefix for the test messages. See the rest of examples and my comment on the same matter above.

That helps to distinguish the messages by their purpose. And avoid confusion in future, when someone creates another `StartProject` message with its specific purpose.

Reply:
I’ve added the prefix to the messages created by me, but I didn’t get to changing all existing messages in this test suite. If you want to I can update them after the PR is accepted, so that refactorings wouldn’t get in the way of the real changes.